source/modules/MEI.text.xml

```diff
@@ -185,8 +185,8 @@
     </remarks>
   </elementSpec>
   <elementSpec ident="l" module="MEI.text">
-    <gloss versionDate="2022-05-18" xml:lang="en">line of text</gloss>
-    <desc xml:lang="en">Contains a single line of text within a line group.</desc>
+    <gloss versionDate="2023-01-26" xml:lang="en">verse line</gloss>
+    <desc versionDate="2023-01-26" xml:lang="en">Contains a single, possibly incomplete, line of verse.</desc>
```

Comment:
I'm OK with the internal move of the element to the text module, but am hesitating to accept this PR because you're also changing the user-facing description, and therefore the semantics of the element itself at the same time. Is there a specific issue you are trying to address here?